src/listpack.c

```diff
@@ -2111,15 +2111,15 @@ int listpackTest(int argc, char *argv[], int flags) {
 
     TEST("Stress with variable listpack size") {
         unsigned long long start = usec();
-        int maxsize = accurate ? 16384 : 16;
+        int maxsize = accurate ? 8192 : 16;
         stress(0,100000,maxsize,256);
         stress(1,100000,maxsize,256);
         printf("Done. usec=%lld\n\n", usec()-start);
     }
 
     /* Benchmarks */
     {
-        int iteration = accurate ? 100000 : 100;
```

Comment:
on a second thought, i think i like this direction even less.
some of these benchmarks run quite fast normally, and trying to compare the effect of a certain change with a low number of iterations will not show anything.
e.g. the lpSeek benchmark takes just 1.7 seconds on my machine.
maybe the better approach for benchmarking and even fuzz testing should be to run the test for a fixed number of seconds, rather than fixed number of iterations, but it makes the benchmark code more complex....

considering my previous post, since ASAN is so much slower, maybe your initial approach was correct.
i just wonder if we should apply that same change (remove the --accurate flag) on other ASAN tests.

Reply:
@oranagra looks like `--accurate` flag in other ASAN tests doesn't cause slow down. What do you expect to see when you remove from all tests?  I'm not sure keeping `--accurate` flag will make it likely to catch issues, maybe it'll never help with anything but we can keep it as long as it doesn't cause an issue.